resources/js/Plugins/date-formatter.js

```diff
@@ -0,0 +1,18 @@
+import moment from "moment";
+
+export function calendar(timestamp){
```

Comment:
Since these are utility functions, maybe a comment above each function describing its purpose would be a good idea

Reply:
Good call, on it